app/src/initializeApp.js

```diff
@@ -37,7 +37,16 @@ function setOptions(urlParams, store) {
   }
 }
 
+export const bootError = (error) => {
+  document.querySelector("#app").innerText = `
```

Comment:
is this really the best way to do this?

Reply:
need to do basic JS here as the whole Vue app might have been crashed so we can't use virtual dom rendering.